src/components/v2/Notice/index.tsx

```diff
@@ -29,22 +29,30 @@ export const Notice = ({ className, title, description, variant = 'info' }: INot
   const styles = useStyles();
 
   return (
-    <Paper css={[styles.root, styles.getRootStyles({ variant })]} className={className}>
-      <Icon
-        css={[styles.icon, styles.getIconStyles({ variant })]}
-        name={getNoticeIconName(variant)}
-      />
-      <div css={styles.content}>
-        {title && (
-          <Typography variant="small2" color="text.primary" css={styles.title}>
-            {title}
-          </Typography>
-        )}
-        <Typography variant="small2" color="text.primary">
-          {description}
-        </Typography>
-      </div>
-    </Paper>
+    <div css={styles.root}>
+      <Paper css={[styles.inner, styles.getInnerStyles({ variant })]} className={className}>
+        <Icon
+          css={[styles.icon, styles.getIconStyles({ variant })]}
+          name={getNoticeIconName(variant)}
+        />
+        <div css={styles.content}>
+          {title && (
+            <Typography variant="small2" color="text.primary" css={styles.title}>
+              {title}
+            </Typography>
+          )}
+          {description && (
+            <Typography
+              variant="small2"
+              color="text.primary"
+              css={styles.getDescription({ hasMarginTop: Boolean(title) })}
```

Comment:
nit: for consistency we should cast booleans the same way I think `!!` is nice shorthand for doing this.